samples/hello_world/src/main.c

```diff
@@ -6,13 +6,40 @@
 
 #include <stdio.h>
 #include <zephyr/kernel.h>
+#include <zephyr/drivers/sensor.h>
```

Comment:
This i will totally remove once we create our own application after porting Virgo.

Reply:
ok, in that case will roll back the generic hello world to what was earlier. Or will you re-merge the old hello world. Because this will include all the earlier merges from us on top of which you will re-merge the original hello world

Reply:
I was thinking about just removing everything from the hello world and copy the original one line in it. But, I understand your point as well that in historical data this will appear. Maybe we can already try to create our own application and do all kinds of experiments there.